pcdsdevices/pseudopos.py

```diff
@@ -2,13 +2,15 @@
 import typing
 
 import numpy as np
+import ophyd
 from ophyd.device import Component as Cpt
 from ophyd.device import FormattedComponent as FCpt
 from ophyd.pseudopos import (PseudoPositioner, PseudoSingle,
                              pseudo_position_argument, real_position_argument)
 from scipy.constants import speed_of_light
 
 from .interface import FltMvInterface
+from .signal import Signal
```

Comment:
Any reason to pull this from local instead of from ophyd?

Reply:
Haven't a clue why I did that; pushed a fix.

Reply:
:+1: